### ISSUE TITLE
model issue

### ISSUE THREAD

**Issue by fenghuoxiguozu** [CLendering/SubspaceAD#6]
<img width="1185" height="514" alt="Image" src="https://github.com/user-attachments/assets/587af132-9752-4c35-a3c3-826f542be45c" />
This project can only be trained for one epoch? Is there no model saving?

**Comment by ZhouH01** [comment]
The methods is train-free, but it's should save the subspace data, you can check this.

**Comment by CLendering** [comment]
Hi! As mentioned, SubspaceAD is a training-free method. It doesn't use backpropagation or gradient descent, so there are no "epochs" in the traditional sense, it just performs a single-pass PCA fit on the features.

Because fitting the PCA on a few-shot set is fast, there is no need to save a model file. The outputs (anomaly maps and results) are saved directly to the `--outdir` you specify.
